example/src/Bar.java

```diff
@@ -1,8 +1,6 @@
 package src;
 
 // Unused imports are suppressed in suppressions.xml, so this should not raise issue.
-import java.util.Objects;
```

Comment:
These were removed when running `bazel run //:format` so im not sure if the supressions.xml are working correctly.

Reply:
I think something about the Java formatter is misbehaving and making non-format changes. Could you just revert the .java changes pls?

Reply:
On it